Library/Homebrew/manpages/brew.1.md

```diff
@@ -451,6 +451,10 @@ With `--verbose` or `-v`, many commands print extra debugging information. Note
 
     If no <formulae> are provided, all linked app will be removed.
 
+  * `unlinkapps --prune`:
+    Removes links created by `brew linkapps` that are no longer valid, e.g.,
+    because a formula was removed or apps provided by a formula have changed.
```

Comment:
This sounds like a better place for this functionality; thanks for pointing it out! Do you mean as default behavior for `brew prune` or as a separate mode of operation, e.g. `brew prune --apps`?